source/localizable/testing/testing-components.md

```diff
@@ -126,7 +125,7 @@ test('should update title on button click', function(assert) {
   assert.equal(this.$('h2').text(), 'Hello World', 'initial text is hello world');
 
   //Click on the button
-  this.$('button').click();
+  document.getElementsByClassName('title-button')[0].click();
```

Comment:
Sticking to pure DOM apis, I think I'd rather recommend `document.querySelector('.title-button').click()`.

However, chances are that if you don't wrap this click like `Ember.run(() => document.querySelector('.title-button').click())` the test suite will complain about "run loop in testing is paused blah blah blah"). Should we mention that? (`ember-native-dom-helpers` wrap everything in `Ember.run` just in case)